### ISSUE TITLE
[UI] Nav bar transparency

### ISSUE THREAD

**Issue by tens0rfl0w** [corona-warn-app/cwa-app-ios#767]
* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug
Just a short one: If you open the FAQ from the home view and close it, the nav bar stays solid and does not go transparent when scrolling back to the top. Sometimes the opposite happens and the navbar stays transparent even when scrolling.

![image](https://user-images.githubusercontent.com/57523343/85552239-17586100-b623-11ea-9dc9-ed7a7ed3feb5.jpeg)
![image](https://user-images.githubusercontent.com/57523343/85552344-33f49900-b623-11ea-8265-ff467bf5dede.png)

---
Internal Tracking ID: EXPOSUREAPP-1944

**Comment by EisvogelSAP** [comment]
Hey @tens0rfl0w, thanks for pointing this out. Will have fix incoming for this tomorrow morning. :-)

**Comment by EisvogelSAP** [comment]
Hey @tens0rfl0w, sorry for the delay here. Unfortunately I was overruled. This needs to be aligned internally before we can continue here.

Again, thanks for raising this issue to us and sorry for the delay.

**Comment by tens0rfl0w** [comment]
Hey @EisvogelSAP, 

I have seen the discussion in the PR. No worries, luckily it breaks no functionality!🙂

**Comment by aleksandr-tikhonov** [comment]
I'd like to provide a comparison of custom and default nav bar implementations

### Custom
on the top
![custom 1](https://user-images.githubusercontent.com/61236601/86588928-f0881c00-bf8c-11ea-9ace-bd41d948ec9e.PNG)
on the middle/bottom
![custom 2](https://user-images.githubusercontent.com/61236601/86588934-f2ea7600-bf8c-11ea-9555-de5f3bbd6cf9.PNG)


### Default
on the top
![default 1](https://user-images.githubusercontent.com/61236601/86588940-f847c080-bf8c-11ea-871f-c56a33886f6d.PNG)
on the middle/bottom
![default 2](https://user-images.githubusercontent.com/61236601/86588947-fbdb4780-bf8c-11ea-97af-df20b9cad4a9.PNG)

I also attached the video with demonstration how it works.
[demo videos.zip](https://github.com/corona-warn-app/cwa-app-ios/files/4878556/demo.videos.zip)

**Comment by tens0rfl0w** [comment]
@aleksandr-tikhonov For me personally the standard approach looks fine, I guess it depends more on the decision of the design team than on how practically both implementations are.

If we are talking about a „redesign“ please also consider [this](https://github.com/corona-warn-app/cwa-wishlist/issues/47) issue from the wishlist repository. I agree with the issue author that on non-notch devices (iPhone 7, 8, SE2, ...) the headroom between the logo and the Status bar is to small.

**Comment by moritzebeling** [comment]
Since you mentioned "design team" and "redesign", I think it would be nice to consider some icon design adjustments as soon as possibly possible. I already opened an issue to collect some ideas: https://github.com/corona-warn-app/cwa-wishlist/issues/78 Feel free to share your designs as well. The app will be on our phones for quite some time, so it would be nice if it would be beautiful.

**Comment by tens0rfl0w** [comment]
Hey @moritzebeling,

I‘ve seen your logo concepts, but please note that this issue is not a enhancement request. Right now we are talking about a re-implementation of the navigation bar due to some bugs that appeared with the custom one. I mentioned the enhancement request for the navigation bar headroom only because it seems like the safe area for the navigation bar is off on some devices.

A logo redesign is out of scope of this issue, but your idea will surely be considered by the team!🙂

**Comment by dsarkar** [comment]
@tens0rfl0w,

Thanks for contributing.
This issue can´t be reproduced on CWA 1.10.1 / iOS 14.3 / iPhone 6s anymore.
The issue seems fixed after major changes in recent releases.

Any objections to closing this issue? 

Best wishes, DS

**Comment by tens0rfl0w** [comment]
Yes, this was fixed a while ago. Closed👍🏼

**Comment by dsarkar** [comment]
@tens0rfl0w Thanks !
